### DISCUSSION TITLE
New Work Modes - Power Station & Peak Shaving

### DISCUSSION THREAD

**Discussion by andrewmooreio** [nathanmarlor/foxess_modbus#385] [General]
Hi All,

I've noticed that two new work modes, Power Station & Peak Shaving, have appeared for the H1 inverter when using FoxESS Cloud. I don't go on the web portal often, so it may have been out for a while. Apologies if so, it's new to me!

![image](https://github.com/nathanmarlor/foxess_modbus/assets/20435317/c764f5cf-4fd4-46e8-a872-cb558e1fdbdd)

Does anyone know what they do and can those work modes be set using this integration?

Many thanks.

**Comment by canton7** [comment]
According to discussion on the Facebook group, noone's entirely sure what these are and they haven't been implemented in the inverters yet.

**Reply by andrewmooreio** [reply]
Thanks. Interesting that they'd make it available via the web app/API before implementing the backend functionality.

Here's me hoping we might get force discharge to grid...

**Reply by canton7** [reply]
Yeah, the same sentiment was on Facebook!

**Reply by FozzieUK** [reply]
From what the FoxESS support engineer said at least one of these options will be country specific (mainland Europe), I had a hint they are considering a possible future option of battery grid export based on the 2 time windows - I assume this will be linked with Octopus Intelligent Octopus Flux as GivEnergy are the only one working with this at the moment.

The latest H1 inverter version doesn't support anything other then the original 3 modes for now.

(sorry forgot to say the 4th mode 'Weakgridadj' is also another country specific thing, some kind of utility control like DRM - it doesn't do anything yet.)

EDIT: Latest version of the app now has a '?' to show the description for each of these new modes - I guess coming soon ?
